src/GafferUI/BackdropNodeGadget.cpp

```diff
@@ -113,8 +113,17 @@ std::string BackdropNodeGadget::getToolTip( const IECore::LineSegment3f &line )
 	}
 
 	const Backdrop *backdrop = static_cast<const Backdrop *>( node() );
-	std::string title = backdrop->titlePlug()->getValue();
-	std::string description = backdrop->descriptionPlug()->getValue();
+	std::string title = "Error evaluating title";
+	std::string description;
+	try
+	{
+		title = backdrop->titlePlug()->getValue();
+		description = backdrop->descriptionPlug()->getValue();
+	}
+	catch( const std::exception &e )
+	{
+		description = std::string( "EVALUATION ERROR:\n" ) + e.what();
```

Comment:
Really minor thing, but let's format this as "ERROR : Getting $whatever" to match the formatting of error messages elsewhere (same as IECore::MessageHandler basically).